src/oldtime.rs

```diff
@@ -132,6 +132,16 @@ impl Duration {
         Duration { secs: secs, nanos: nanos as i32 }
     }
 
+    /// Makes a new `Duration` with the given number of seconds and nanoseconds.
+    pub fn secs_nanos(secs: i64, nanos: i32) -> Duration {
```

Comment:
This should probably be marked inline so that it's available for inlining across crate boundaries, otherwise since there are no generics rustc will generate the object code ahead of time.

Also since this is the first multi-parameter constructor for duration I think it could be named `seconds_nanos`, to keep it in-line with the single-parameter constructor naming.

Could this also document that if the number of nanoseconds is greater than 1B that the nanos will carry over into the seconds?